docs/_config.yml

```diff
@@ -31,34 +28,68 @@ kramdown:
   input: GFM
   hard_wrap: false
 
-theme: minima
+sass:
+  style: compressed
+
+# theme: minima
+# End
+
+# Gem settings
 gems:
   - jekyll-feed
   - jekyll-seo-tag
   - jekyll-sitemap
   - jekyll-avatar
+  - jekyll-github-metadata
+
 exclude:
   - Gemfile
   - Gemfile.lock
+  - README.md
+  - SITECHANGELOG.md
+# End
+
+# Github metadata settings
+repository: MvvmCross/MvvmCross
+# End
 
-# jekyll-seo-tag config
+# jekyll-seo-tag settings
 social:
   name: MvvmCross
   links:
     - https://twitter.com/MvvmCross
     - https://github.com/MvvmCross
 
-logo: /img/MvvmCross-avatar.png
+logo: /assets/img/logo/MvvmCross-avatar.png
 
 twitter:
   username: MvvmCross
-  
-permalink: /:title/
```

Comment:
Please don't change appearance of permalinks. We just changed it 2 days ago. Changing back and forth derps up peoples current urls they might be using.